Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+
+
+namespace BootCamp.Chapter
+{    public class Lesson3
+    {   public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                Console.Clear();
+                GatherData(out string firstName, out string lastName, out int age, out float weight, out float height);
+                DisplayOutput(firstName, lastName, age, weight, height);
+            }
+        }
+        public static string GetStringInput(string prompt)
+        {
+            Console.WriteLine(prompt);
+            string value = Console.ReadLine();
+            
+            if (Empty(value))
+            {
+                Console.Write("Name cannot be empty.");
+                return "-";
+            }
+
+            //default case
+            return value;
+        }
+        public static int GetIntInput(string prompt)
+        {
+            Console.WriteLine(prompt);
+            string value = Console.ReadLine();
+
+            if (Empty(value))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886326)